.eslintrc

```diff
@@ -0,0 +1,32 @@
+{
+    "env": {
+        "browser": true,
+        "es2021": true,
+        "node": true
+    },
+    "extends": ["eslint:recommended", "prettier"],
```

Comment:
Did you consider extending `@nahmii` (from github:nahmii/eslint-config) rather than `eslint:recommended`?

Reply:
Ran into some issues here. Had to explicitly import `minimatch` or some weird reason. 

Latest commits make it work though. It uses the Nahmii style guide and prettier to format everything properly.